router-sink/src/main/java/org/springframework/cloud/stream/module/router/sink/RouterSinkProperties.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.module.router.sink;
+
+import javax.validation.constraints.AssertTrue;
+import javax.validation.constraints.NotNull;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.core.io.Resource;
+import org.springframework.expression.Expression;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
+
+/**
+ * Properties for the Router Sink; the router can use an expression
+ * or groovy script to return either a channel name, or a key to
+ * the channel mappings map.
+ *
+ * @author Gary Russell
+ *
+ */
+@ConfigurationProperties
+public class RouterSinkProperties {
+
+	public static final Expression DEFAULT_EXPRESSION =
+			new SpelExpressionParser().parseExpression("headers['routeTo']");
```

Comment:
I don't know if it feasible for SCS-Modules, but this one would be faster:
````java
private Expression expression = new FunctionExpression<>(new Function<Message<?>, Object>() {

	@Override
	public Object apply(Message<?> message) {
		return message.getHeaders().get("routeTo");
	}
		
});
````
bypassing any SpEL conversion, instantiating performance impact.